shared/wallets/confirm-form/participants/container.js

```diff
@@ -17,6 +17,7 @@ const mapStateToProps = (state: TypedState) => {
     fromAccountName: '',
     fromAccountContents: '',
     recipientUsername: built.toUsername,
+    recipientFullName: 'placeholder', // TODO: get a way to get full names from usernames
```

Comment:
One issue I kept running into during this PR was receiving a user's full name, as in the mockups. It'd be really nice to have a user's full name returned by the built payment RPC, but I'm not sure if there'd be a better place to get that info?